src/Core_Fix_ResourceUnloadOptimizations/Core.ResourceUnloadOptimizations.cs

```diff
@@ -89,9 +110,9 @@ private static bool PlentyOfMemory()
             if (mem == null) return false;
 
             // Clean up more aggresively during loading, less aggresively during gameplay
-            var isLoading = GetIsNowLoadingFade();
+            var isLoading = IsLoadingChecks.Any(x => x());
```

Comment:
Lambda check is slower than an `if (Constants.InsideStudio)` while needing more code.

Reply:
removed